tailwind.config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   darkMode: false, // or 'media' or 'class'
   theme: {
     fontFamily: {
+      'kumbhsans': ['Kumbh Sans', 'sans-serif'],
```

Comment:
Here we should only use one of them `kumbhsans` or `head` since both have the same content. I'd recommend using `head` since it's not bound the it's actual font and can in the future be replaced without changing the variable name.